Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/FleetDriverAssociation.yaml

```diff
@@ -27,6 +27,10 @@ FleetDriverAssociation:
     deleteByDriverId:
       kvFunction: deleteWithKV
       where: driverId
+    findByFleetIdAndDriverId:
```

Comment:
I guess `Storage.Queries.FleetDriverAssociationExtra.findByDriverIdAndFleetOwnerId` is the same